omnibus/resources/agent/msi/cal/strings.cpp

```diff
@@ -231,9 +231,6 @@ bool loadPropertyString(MSIHANDLE hInstall, LPCWSTR propertyName, wchar_t **dst,
     *dst=szValueBuf;
     *len = cchValueBuf;
     toMbcs(propval, szValueBuf);
-    WcaLog(LOGMSG_STANDARD, "loaded property %s = %s", propertyname.c_str(), propval.c_str());
```

Comment:
should we still log the `propertyname`, or is that also sensitive?

Reply:
the property name isn't sensitive, but it's probably not all that helpful.